app/scripts/contentscript.js

```diff
@@ -546,7 +541,7 @@ const start = () => {
 
   if (shouldInjectProvider()) {
     if (!isManifestV3) {
-      injectScript(inpageBundle);
+      injectScript(chrome.extension.getURL('inpage.js'))
```

Comment:
Shouldn't this part be more like [this](https://github.com/MetaMask/metamask-extension/blob/develop/app/scripts/platforms/extension.js#L82) instead of using `chrome`?